rust-toolchain

```diff
@@ -0,0 +1 @@
+stable
```

Comment:
I'd much prefer to have this removed too.  We don't really need it I believe.  If we want to keep the Makefile, it can do without specifying the toolchain, and latest stable will be chosen.